docs/STORY-BEHIND.md

```diff
@@ -21,11 +29,15 @@ Working together with people they have never met in their field, they have devel
 ![afetharita](./images/afetharita3.png)
 ![afetharita](./images/afetharita4.png)
 
-**In just 2 days.**
-It is impossible to express how much stress, busyness they have experienced.
-Each of them is a hero.
```

Comment:
This was not a paragraph itself, but I am still open to merge this PR if you could change "## In just 2 days"

Thanks for you effort!

Reply:
@eraygundogmus within a new commit for the same branch, I've removed the title and update the text with new suggestion.